### ISSUE TITLE
Error during manual or schedule run (not by webhook)

### ISSUE THREAD

**Issue by alphatroya** [bitrise-steplib/steps-github-status#9]
If I run build by "Start/Schedule a Build" button I get next error and all workflow fail
```
#<RuntimeError: No commit hash specified>
```
I think it should be an option for not crush build in case of error.
Or maybe you can provide the workaround for this case.

**Comment by viktorbenei** [comment]
Good catch!

We'll discuss what the best solution would be (feel free to suggest a solution!), for now you can "fix" this by specifying a `run_if` for the step:

```
run_if: not (enveq "BITRISE_GIT_COMMIT" "")
```

This will skip the step if `BITRISE_GIT_COMMIT` is empty / not defined. You can find more info about `run_if` at http://devcenter.bitrise.io/tips-and-tricks/disable-a-step-by-condition/

**Comment by viktorbenei** [comment]
Should we change the default `run_if` of this step so that it only runs if there's a Commit Hash? WDYT?

**Comment by alphatroya** [comment]
Looks like `run_if` option solve the problem, thanks. But I still think that it should be a default behaviour.

**Comment by viktorbenei** [comment]
Definitely! It's on our task list, for one more quick discussion I believe, but so far we discussed other possible solutions and only the `run_if` one seems to be "robust enough" (e.g. that it works even if there's no Git Clone step in the workflow or the build fails before or at Git Clone).

**Comment by kdobmayer** [comment]
Hi @alphatroya ,

We released a new version ([1.1.1](https://github.com/bitrise-steplib/steps-github-status/releases/tag/1.1.1)) for the step. We made the step skippable, which means the step not failing anymore if the commit hash is not specified, but shows a warning message: “GitHub requires a commit hash for build status reporting”.

Thanks again for reporting the issue!
